internal/ignition/ignition_test.go

```
@@ -119,3 +119,103 @@ var _ = Describe("Bootstrap Ignition Update", func() {
+1XWdHQ==
+-----END CERTIFICATE-----`
+
+	var (
+		err        error
+		masterPath string
+		workerPath string
+		caCertPath string
+		file       *config_31_types.File
+	)
+
+	BeforeEach(func() {
+		var err1 error
+		masterPath = filepath.Join(workDir, "master.ign")
+		workerPath = filepath.Join(workDir, "worker.ign")
+		err1 = ioutil.WriteFile(masterPath, []byte(ignition), 0600)
+		Expect(err1).NotTo(HaveOccurred())
+		err1 = ioutil.WriteFile(workerPath, []byte(ignition), 0600)
+		Expect(err1).NotTo(HaveOccurred())
+
+		caCertPath = filepath.Join(workDir, "service-ca-cert.crt")
+		err1 = ioutil.WriteFile(caCertPath, []byte(caCert), 0600)
+		Expect(err1).NotTo(HaveOccurred())
+	})
+
+	Describe("update ignitions", func() {
+		Context("with ca cert file", func() {
+			It("got ca cert added to ignitions", func() {
+				g := NewGenerator(workDir, installerCacheDir, cluster, "", caCertPath, log).(*installerGenerator)
+				err = g.updateIgnitions()
+
+				masterBytes, _ := ioutil.ReadFile(masterPath)
+				masterConfig, _, err1 := config_31.Parse(masterBytes)
+				Expect(err1).NotTo(HaveOccurred())
+				Expect(masterConfig.Storage.Files).To(HaveLen(1))
+				file = &masterConfig.Storage.Files[0]
+				Expect(err).NotTo(HaveOccurred())
+				Expect(file.Path).To(Equal(common.ServiceCACert))
+
+				workerBytes, _ := ioutil.ReadFile(workerPath)
+				workerConfig, _, err1 := config_31.Parse(workerBytes)
+				Expect(err1).NotTo(HaveOccurred())
+				Expect(workerConfig.Storage.Files).To(HaveLen(1))
+				file = &masterConfig.Storage.Files[0]
+				Expect(err).NotTo(HaveOccurred())
+				Expect(file.Path).To(Equal(common.ServiceCACert))
+			})
+		})
+		Context("with no ca cert file", func() {
+			It("got no ca cert added to ignitions", func() {
+				caCertPath = ""
+				g := NewGenerator(workDir, installerCacheDir, cluster, "", caCertPath, log).(*installerGenerator)
+				err = g.updateIgnitions()
+
+				masterBytes, _ := ioutil.ReadFile(masterPath)
```

Don't ignore errors. It looks like this is in a few places.